### ISSUE TITLE
Support for Navigation Drawer

### ISSUE THREAD

**Issue by anandwana001** [robolectric/robolectric#5954]
### Description
To test NavigationDrawer we need some support for delay using which we can delay till the navigation drawer opens up after clicking of the hamburger menu. 

### Steps to Reproduce
```
onView(withContentDescription(R.string.drawer_open_content_description)).perform(click())
onView(withId(R.id.administrator_controls_linear_layout))
        .perform(nestedScrollTo())
        .check(matches(isDisplayed())).perform(click())
```

### Robolectric & Android Version
Robolectric - 4.3
Android - API 28

**Comment by hoisie** [comment]
Can you provide a more complete example, such as a small Android app and test on Github that demonstrates the issue? I am not sure exactly what the problem is.

**Comment by anandwana001** [comment]
@hoisie Please check the file [here](https://github.com/oppia/oppia-android/blob/bf29565a61880f881001effa7bce590d6d7bf2ce/app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerTestActivityTest.kt#L333)

**Comment by hoisie** [comment]
Please provide clear instructions on how to repro the issue, with expected
results vs what you are seeing. See
https://github.com/robolectric/robolectric/wiki/How-to-Report-Issues. I
don't have much context into that code you linked.

**Comment by anandwana001** [comment]
Describing the issue I was facing earlier is, if you are writing a `sharedTest` and the test says to open the navigation drawer by clicking over the hamburger menu icon, robolectric will not open the drawer. 
So, the workaround here is to see how does it happen in Espresso and we get to know that espresso automatically calls the `computeScroll` function of DrawerLayout at every draw, but the robolectric seems to not doing it, so we called it automatically. 

Workaround PR - https://github.com/oppia/oppia-android/pull/2204

Closing this issue and @hoisie Thanks for sharing the wiki, I will use it in the future.
